### ISSUE TITLE
Exporting spreadsheet to CSV change the format of the dates

### ISSUE THREAD

**Issue by jpberdejo** [ONLYOFFICE/DocumentServer#2629]
### This issue is unique.

- [X] I have used the [search tool](https://github.com/ONLYOFFICE/DesktopEditors/issues?q=) and did not find an issue describing my bug.

### Operating System

Windows 10 / 8.1 / 7

### Version information

8.0.1

### Expected Behavior

When saving a spreadsheet into CSV file, expect the dates keep the same actual format.

### Actual Behavior

When saving a spreadsheet into CSV file, the dates formatted as dd/mm/yyyy are changed to dd.mm.yyyy what is not valid when import the file in another app.

### Reproduction Steps

1. Open or create a spreadsheet file with dates formatted as dd/mm/yyyy.
2. Save as CSV file.
3. Open CSV saved file in any text editor.
4. Check the dates.

**Comment by ZachreyNM** [comment]
I have the reverse problem! I am trying to IMPORT a comma delimited csv with correct dates viewed in an ascii editor but in the format yyyy-mm-dd. Upon importing as comma delimited, UTF-8 file, the dates are really strange.

"Account Name","Processed Date","Description","Check Number","Credit or Debit","Amount"
"checking",2023-12-29,"desc esth rt srtnsgf1",,"Debit",9.99
"checking",2023-12-26,"desc esth rt srtnsgf2",,"Credit",1.99
"checking",2023-12-21,"desc esth rt srtnsgf3",,"Credit",1.99

When I open it in OnlyOffice I get this:

Account Name Processed Date Description Check Number Credit or Debit Amount
checking 26.03.7369 desc esth rt srtnsgf1 Debit 9.99
checking 30.05.7368 desc esth rt srtnsgf2 Credit 1.99
checking 16.01.7367 desc esth rt srtnsgf3 Debit 1.99

Any suggestions on how to correct this problem?

THANKS!!

**Comment by bigbeka** [comment]
> The issue is actual in DocumentServer.

No, it is not in the server. I am getting this issue in the MacOS app when opening a CSV file.

**Comment by bigbeka** [comment]
> Hello! @ZachreyNM, @bigbeka Instead of just opening the CSV/TXT file, try to extract the data from it. To do this, open the spreadsheet editor. Go to the Data tab and use the 'Get data' function.

Other spreadsheet editors don't do that when opening a CSV, plus when importing data from CSV with "Get data" function, it imports the Quotations, see screenshot.
I would say opening a CSV is fairly common use case as it is opening XLSX files instead of "Get data" every time.


<img width="222" alt="Screenshot 2024-04-24 at 14 50 51" src="https://github.com/ONLYOFFICE/DocumentServer/assets/53529055/18cc23c2-8e71-40e7-86e1-3d80db72e435">
<img width="248" alt="Screenshot 2024-04-24 at 14 51 02" src="https://github.com/ONLYOFFICE/DocumentServer/assets/53529055/273a3cb9-34cb-4e1e-ada3-46e4efc076f7">

**Comment by alinasafonova** [comment]
Hello, @jpberdejo
This issue will be fixed in the next release.

**Comment by Rita-Bubnova** [comment]
DocumentServer v8.3.0 is released so I close this issue. Feel free to comment or reopen it if you got further questions.

**Comment by Korne127** [comment]
Hey @Rita-Bubnova 
I found this issue by wanting to report the same bug
It still exists, just the other way around: When I create a spreadsheet and write a date like `20.12.2024` (which is the default German locale format), OnlyOffice of course recognizes it as a date. But when exporting it to a CSV file, it replaces those dots with slashes.
This is very weird, as the date format gets arbitrarily changed. I would understand it if it would get changed into a standardized format like YYYY-MM-DD, but it changes it into MM/DD/YYYY, which is extremely weird, as slashes are usually used the other way around, so this is just asking for errors to be made when processing the CSV.
I'll assume this error somehow got included while trying to fix this issue, but just replacing dots with slashes does not work at all. The format exported to the CSV should not be changed; it should just be the one displayed to the user. Not changed from slashes to dots like originally described, but not changed the other way around either.
I hope that this gets fixed soon, so that the CSV export actually contains the correct dates in the correct format, as they should be.
